server/views/pages/curated-lists.njk

```diff
@@ -63,8 +63,21 @@
           </div>
         </div>
       {% else %}
-        {# Generic series #}
-        {% componentV2 'series-container', section.value %}
+        <div class="row {{ {s:2} | spacingClasses({padding:['top']}) }} {{ {s:4} | spacingClasses({margin: ['bottom']}) }} relative">
```

Comment:
Does this remove the need for the `series-container` component?

Reply:
It was in our period of buildeverythingbecausewewillneeditinthefuture phase.
Removed.